src/dashboard/trade/placeOrder/AmountDetails.tsx

```diff
@@ -21,12 +21,72 @@ type AmountDetailsProps = {
 const useStyles = makeStyles(() => {
   return createStyles({
     iconContainer: {
+      color: "#979797",
       display: "flex",
       alignItems: "center",
+      fontSize: "16px",
+    },
+    text: {
+      color: "#979797",
+      fontSize: "16px",
```

Comment:
It seems we could make use of text components that have sizes, colors, etc. Would eliminate duplicate code and make possible restyling and maintaining lot easier.